src/main/java/hudson/plugins/brakeman/scanners/BrakemanJSONScanner.java

```diff
@@ -0,0 +1,84 @@
+package hudson.plugins.brakeman.scanners;
+
+import hudson.plugins.analysis.core.ParserResult;
+import hudson.plugins.analysis.util.model.Priority;
+import hudson.plugins.brakeman.Warning;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * A Java class that represents a JSON Scanner of the Brakeman Output file
+ * This class includes a specific strategy for implementing the parsing process
+ * of the output file.
+ *
+ * @author Thomas Baird
+ */
+public class BrakemanJSONScanner implements AbstractBrakemanScanner {
+    /**
+     * Creates a new instance of <code>BrakemanJSONScanner</code>
+     */
+    public BrakemanJSONScanner() {}
+
+    public void scan(String content, ParserResult project) {
+        this.scanWarnings(content, project);
+        this.scanIgnoredWarnings(content, project);
+    }
+
+    private void scanWarnings(String brakemanOutput, ParserResult project) {
+        try {
+            JSONObject brakemanFile = new JSONObject(brakemanOutput);
+            scanJSONResultSet(brakemanFile, project, "warnings");
+        } catch(JSONException e) {
+            e.printStackTrace();
```

Comment:
Yeah so this is exactly what I need to do. I need the plugin logger in this and I need to log some message to it. I need to figure out how get the logger though